packages/go/build.mjs

```diff
@@ -0,0 +1,3 @@
+import { esbuild } from '../../utils/build.mjs';
+
+await esbuild({ bundle: true, external: ['@vercel/build-utils'] });
```

Comment:
Should also mark `dependencies` as external

Reply:
Non issue now